src/components/ListItem.jsx

```diff
@@ -71,6 +84,7 @@ export function ListItem({ name, items, listToken }) {
 					onChange={handleCheckbox}
 					defaultChecked={isChecked}
 					disabled={isDisabled}
+					aria-label={handleAriaForColors}
```

Comment:
This doesn't render in the browser HTML.

Reply:
This does not render in the HTML but will only be read aloud by a screen reader for accessibility: so that a visually impaired user can potentially know what the colors are for

Reply:
If this code words, then I expect to see that attribute in the HTML rendered in the browser. Here's an example of how this would render if I have specified aria-label:

<img width="454" alt="Screenshot 2022-11-21 at 20 28 41" src="https://user-images.githubusercontent.com/4036118/203142434-09081bfe-1976-4fb0-a483-1c0836d349d0.png">

However, this is what is rendered:

<img width="544" alt="Screenshot 2022-11-21 at 20 24 49" src="https://user-images.githubusercontent.com/4036118/203142603-c1d89dd7-c946-41e9-845e-f25893008904.png">

Why isn't the `aria-label` attribute displayed for the <input />? How would screen readers pick it up if it's not added to the HTML attributes?

Reply:
Hi @pmbanugo I've removed this part of the code and will focus on CSS this week since we cannot move forward unless we merge. If we have more time afterwards, I'll be happy to revisit 😅